test/test_utils.py

```diff
@@ -1,5 +1,8 @@
+import boto3
+import botocore.session
 import pytest
 
+from botocore.stub import Stubber
```

Comment:
where is this used? I'm unsure how these tests are isolated from calling to AWS (and we do want to ensure that these are isolated and mocking AWS).

Reply:
not sure where, so i just got rid of them 🤷‍♂️